Bullwinkle.class.nut

```diff
@@ -147,7 +151,6 @@ class Bullwinkle {
             "name": command,
             "data": data,
             "ts": ts,
-            "tries": 0,
```

Comment:
I should have better stated that I don't expect anyone would be using those fields on the _partner side (although maybe for analytics on the number of failures?).  

My opinion is that this data is wasteful and doesn't need to be transmitted over the wire but there may be use cases I haven't thought of as I didn't write the lib originally...

I also feel that way about message.retry, however, I haven't gotten around to making that change based on the push back in the PR so far.